include/SYCL/sycl.hpp

```diff
@@ -0,0 +1 @@
+#include "../CL/sycl.hpp"
```

Comment:
Do not add this.

It is a ComputeCpp bug which has been fixed long time ago.
Make a PR against Eigen & TensorFlow instead. :-)